TurkishFilter/sections/css_extended.txt

```diff
@@ -18,6 +18,7 @@ star.com.tr##li:has(> a[href*="//apps.star.com.tr/advredir"])
 gazeteoku.tv##td[valign="top"]:has(table[cellspacing="1"] b:contains(Reklamlar))
 cimri.com##div[name="teklifler"] > div[class] > table[class] > tbody > tr:has(> td > div[class] > div[id*="_728x90_"])
 cimri.com##div[name="teklifler"] > div[class] > section[class]:has(> a[href*="&utm_campaign=cimri_sponsor_text_link"][rel="nofollow"])
+aksam.com.tr#?#.column > .center:has(> .wrapper)
```

Comment:
.wrapper is too generic. Better > div > div[id^="div-gpt-ad-"].

Mobile app:
#bottomrightside > a[href*="/app"]